web/templates/index.html

```diff
@@ -218,7 +218,7 @@ <h3 class="card-title">Filters</h3>
                             alt="A drawing of an empty box." />
                     </div>
                     <div id="library-item" style="display: none;" class="list-group-item library-item">
-                        <input hidden type="text" class="library-item-id" value="" aria-hidden="true" />
+                        <input hidden type="hidden" class="library-item-id" value="" />
```

Comment:
These changes are semantic. These are inputs that are being hidden using CSS, which is visually fine, but are actually still being shown to blind users. These should be of **type** `hidden` instead, since _semantically,_ they are not supossed to be seen.